packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -15,6 +15,14 @@ const FeedbackStore = types
     onHide: () => true
   }))
   .views(self => ({
+    get applicableRules () {
+      return flatten(Array.from(self.rules.values()))
+        .map(rule => {
+          if ((rule.success && rule.successEnabled) || (!rule.success && rule.failureEnabled)) {
```

Comment:
I might be remembering this wrongly, but aren't the rules already filtered for this when they're first set up?

Reply:
Also, this is better done with `.filter` rather than `.map`.